vtStor.ATest/main.cpp

```diff
@@ -57,14 +70,32 @@ void main()
         // Register command handler
         drives[1]->RegisterCommandHandler(vtStor::cAta::s_DefaultCommandHandlerCommandType, commandHandlerAta);
 
-        // Initialize and add more protocol, commandhandler here 
+        // Initialize and add more protocol, commandhandler here    
+
+        // Call command
+        vtStor::Ata::IssueCommand_IdentifyDevice(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer);
+        //vtStor::Ata::IssueCommand_ReadDma(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer, 10, 1);
+        //vtStor::Ata::IssueCommand_ReadBuffer(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer);
+        //vtStor::Ata::IssueCommand_Smart(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer, 208);
     }
 
-    // Call command
-    vtStor::Ata::IssueCommand_IdentifyDevice(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer);
-    //vtStor::Ata::IssueCommand_ReadDma(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer, 10, 1);
-    //vtStor::Ata::IssueCommand_ReadBuffer(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer);
-    //vtStor::Ata::IssueCommand_Smart(drives[1], vtStor::cAta::s_DefaultCommandHandlerCommandType, dataBuffer, 208);
+    if (vtStor::eBusType::Scsi == drives[1]->GetBusType())
```

Comment:
If we want to use this as example, should this be else if?
If a drive could have both bus, which is impossible, but assuming it could hypothetically, then the s_DefaultcommandHandlerCommandType must not be the same for the same drive for Ata and Scsi bus.